chainer/functions/connection/multihead_attention.py

```diff
@@ -0,0 +1,297 @@
+import typing as tp  # NOQA
+
+import chainer
+from chainer.functions.activation import softmax
+from chainer.functions.array import concat
+from chainer.functions.array import expand_dims
+from chainer.functions.array import split_axis
+from chainer.functions.array import repeat
+from chainer.functions.array import reshape
+from chainer.functions.array import transpose
+from chainer.functions.array import where
+from chainer.functions.math import average
+from chainer.functions.math import matmul
+from chainer.functions.noise import dropout
+from chainer.functions.connection import linear
+from chainer import types  # NOQA
```

Comment:
It seems that NOQA is no more required! :beer:

Reply:
Thank you 👍